src/cli/record/export.ts

```diff
@@ -20,21 +20,21 @@ const builder = (args: yargs.Argv) =>
       requiresArg: true,
     })
     .option("username", {
-      alias: "u",
       describe: "Kintone Username",
       default: process.env.KINTONE_USERNAME,
       defaultDescription: "KINTONE_USERNAME",
       type: "string",
       requiresArg: true,
     })
+    .alias("u", "username")
```

Comment:
I think it's better to add `TODO:` comments to WA.
e.g. `TODO: remove this workaround after https://github.com/yargs/yargs/issues/2268 is fixed.`